openedx/core/djangoapps/dark_lang/middleware.py

```diff
@@ -154,7 +171,18 @@ def _activate_preview_language(self, request):
 
         # User doesn't have a dark lang preference, so just return
         if not preview_lang:
+            response.delete_cookie(
+                settings.LANGUAGE_COOKIE_NAME,
+                domain=settings.SESSION_COOKIE_DOMAIN
+            )
```

Comment:
I don't think we want to delete the user's language cookie just because they don't have a dark lang preference.

Reply:
Fixed